commit db06ef3331612c0cc48413e57bbe73ef484e251b (refs/changes/48/1948/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-05-25T22:37:32+00:00 (2 years, 4 months ago)
    
    DAPI-340 Allow VMs to provide NIC tag alternatives
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/lib/dapi/index.js b/lib/dapi/index.js
index 8076ea1..c1deb92 100644
--- a/lib/dapi/index.js
+++ b/lib/dapi/index.js
@@ -1,11 +1,13 @@
 /*
- * Copyright 2014 Joyent, Inc.  All rights reserved.
+ * Copyright 2017 Joyent, Inc.  All rights reserved.
  */
 
+var s_dapi_nic_tag_reqs = require('./nic-tags.js');
 var s_dapi_traits = require('./traits.js');
 var s_dapi_server = require('./server.js');
 
 module.exports = {
+	nic_tag_reqs: s_dapi_nic_tag_reqs,
 	traits: s_dapi_traits,
 	server: s_dapi_server
 };
diff --git a/lib/dapi/nic-tags.js b/lib/dapi/nic-tags.js
new file mode 100644
index 0000000..43efc4d
--- /dev/null
+++ b/lib/dapi/nic-tags.js
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2017 Joyent, Inc.  All rights reserved.
+ */
+
+var s_prim = require('../primitive.js');
+var base = s_prim.base;
+
+var s_nic_tag_reqs = {
+	type: 'array',
+	items: base.arr.str.nonempty.opt
+};
+
+module.exports = s_nic_tag_reqs;
